HW5/HW_5_Task_3.py

```diff
@@ -0,0 +1,26 @@
+'''
+✔ Создайте функцию генератор чисел Фибоначчи (см. Википедию).
+'''
+
+
+def gen_fibo(num):
+    if num == 0:
+        yield 0
+    f1 = 0
+    f2 = 1
+    yield 0
+    yield 1
+    for _ in range(num-2):
+        f3 = f1 + f2
+        f1 = f2
+        f2 = f3
+        yield f3
```

Comment:
Позвольте я немного поправлю, но обязательно разберитесь в моих исправлениях
```suggestion
    f1,  f2 = 0, 1
    for _ in range(num):
        yield f1
        f1,  f2 = f2, f1 + f2
```